web-frontend/src/const/assets/assets.ts

```diff
@@ -12,7 +12,7 @@ export interface ITokenData {
   onDelete?: () => void;
 }
 
-export interface DeleteAssetsErrors {
+export interface IDeleteAssetsErrors {
```

Comment:
it's not a good practice it prefix the interface names with I for many reason. I recommend you either find a better name, either leave DeleteAssetsErrors